src/metabase/embedding/api/common.clj

```diff
@@ -332,7 +332,8 @@
                                                         (contains? param-ids-to-remove parameter_id)) param-mappings))))]
     (-> dashboard
         (update :dashcards #(map remove-parameters %))
-        (update :param_fields #(apply dissoc % field-ids-to-remove)))))
+        ;; TODO cleanup
```

Comment:
Still TODO? This one feels like it needs more context to be actionable (or else resolved/dropped in this PR)

Reply:
Fixed